<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>793</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_26">
    <item row="2" column="1">
     <widget class="QDockWidget" name="cavity_drift_measurement_widget">
      <property name="minimumSize">
       <size>
        <width>359</width>
        <height>288</height>
       </size>
      </property>
      <property name="windowTitle">
       <string>Cavity drift measurement</string>
      </property>
      <widget class="QWidget" name="dockWidgetContents_10">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="cavity_drift_start_pushButton">
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cavity_drift_stop_pushButton">
            <property name="text">
             <string>Stop</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_26">
            <property name="text">
             <string>Measurement time [min]</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="cavity_drift_measurement_time_spinBox">
            <property name="maximum">
             <number>10000</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_27">
            <property name="text">
             <string>Number of acquisitions [/ min]</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="cavity_drift_acquisition_rate_spinBox">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <layout class="QGridLayout" name="gridLayout_31">
      <item row="0" column="0">
       <widget class="QDockWidget" name="spectrometer_control_dockWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="windowTitle">
         <string>Spectrometer</string>
        </property>
        <widget class="QWidget" name="dockWidgetContents_2">
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_30">
            <item>
             <widget class="PlotWidget" name="spectrum_2d_PlotWidget"/>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="3" column="0">
               <widget class="ScienDSpinBox" name="spectrum_2d_cb_low_percentile_DoubleSpinBox">
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
                <property name="suffix">
                 <string>%</string>
                </property>
                <property name="minimum">
                 <double>0.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>100.000000000000000</double>
                </property>
                <property name="value">
                 <double>1.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="ScienDSpinBox" name="spectrum_2d_cb_min_DoubleSpinBox">
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
                <property name="suffix">
                 <string>c/s</string>
                </property>
                <property name="minimum">
                 <double>-10.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>1000000000.000000000000000</double>
                </property>
                <property name="value">
                 <double>1000.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="6" column="0">
               <widget class="QPushButton" name="spectrum_2d_save_data_pushButton">
                <property name="text">
                 <string>Save Data</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="ScienDSpinBox" name="spectrum_2d_cb_max_DoubleSpinBox">
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
                <property name="suffix">
                 <string>c/s</string>
                </property>
                <property name="minimum">
                 <double>-10.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>1000000000.000000000000000</double>
                </property>
                <property name="value">
                 <double>100000.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="ScienDSpinBox" name="spectrum_2d_cb_high_percentile_DoubleSpinBox">
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
                <property name="suffix">
                 <string>%</string>
                </property>
                <property name="minimum">
                 <double>0.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>100.000000000000000</double>
                </property>
                <property name="value">
                 <double>100.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="PlotWidget" name="spectrum_2d_cb_PlotWidget">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>80</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="2">
     <widget class="QDockWidget" name="sweep_parameters">
      <property name="windowTitle">
       <string>Non linearity measurement</string>
      </property>
      <widget class="QWidget" name="dockWidgetContents_3">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Start voltage [V]</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="start_voltage_doubleSpinBox">
            <property name="minimum">
             <double>-10.000000000000000</double>
            </property>
            <property name="maximum">
             <double>10.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Stop voltage [V]</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="stop_voltage_doubleSpinBox">
            <property name="minimum">
             <double>-10.000000000000000</double>
            </property>
            <property name="maximum">
             <double>10.000000000000000</double>
            </property>
            <property name="value">
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Number of steps</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="number_of_steps_doubleSpinBox">
            <property name="decimals">
             <number>0</number>
            </property>
            <property name="minimum">
             <double>1.000000000000000</double>
            </property>
            <property name="maximum">
             <double>1000.000000000000000</double>
            </property>
            <property name="value">
             <double>10.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="nl_start_pushButton">
            <property name="text">
             <string>start</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="nl_stop_pushButton">
            <property name="text">
             <string>stop</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="nl_continue_pushButton">
            <property name="text">
             <string>continue</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>646</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
  <customwidget>
   <class>ScienDSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qtwidgets.scientific_spinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
